lib/Extension/Navigation/NavigationExtension.php

```diff
@@ -39,19 +43,31 @@ public function configure(Resolver $schema): void
 
     private function registerPathFinder(ContainerBuilder $container): void
     {
-        $container->register(self::SERVICE_PATH_FINDER, function (Container $container) {
-            return PathFinder::fromDestinations($container->getParameter(self::PATH_FINDER_DESTINATIONS));
-        });
+        $container->register(
+            self::SERVICE_PATH_FINDER,
+            fn (Container $container) => PathFinder::fromAbsoluteDestinations(
+                $container->parameter(FilePathResolverExtension::PARAM_PROJECT_ROOT)->string(),
+                $container->parameter(self::PATH_FINDER_DESTINATIONS)->value() // @phpstan-ignore argument.type
+            )
+        );
 
         $container->register('application.navigator', function (Container $container) {
             return new Navigator(
                 $container->get('navigation.navigator.chain'),
                 $container->get('application.class_new'),
-                $container->getParameter(self::NAVIGATOR_AUTOCREATE)
+                $container->parameter(self::NAVIGATOR_AUTOCREATE)->value(),
+                FilePath::fromString($this->projectRoot($container))
```

Comment:
I think it wouild be sufficient to retrieve the `$container->parameter(FilePathResolverExtension::PARAM_PROJECT_ROOT)->string()` value here, there's no need to process it.

Reply:
When removed processing, tests fails as project root parameter is simply `%project_root%`

Reply:
I think because you're using the wrong parameter? see for example: https://github.com/phpactor/phpactor/blob/0ef0279028d8591d9039150dfe90151a6b579eb5/lib/Extension/Configuration/ConfigurationExtension.php#L99

Reply:
It worked, thanks